models/oci/error.go

```diff
@@ -7,14 +7,21 @@ import (
 )
 
 var (
-	ErrAppendingLayerCode       = "meshkit-11147"
-	ErrReadingFileCode          = "meshkit-11148"
-	ErrUnSupportedLayerTypeCode = "meshkit-11149"
-	ErrGettingLayerCode         = "meshkit-11150"
-	ErrCompressingLayerCode     = "meshkit-11151"
-	ErrUnTaringLayerCode        = "meshkit-11152"
-	ErrGettingImageCode         = "meshkit-11153"
-	ErrValidatingImageCode      = "meshkit-11154"
+	ErrAppendingLayerCode           = "meshkit-11147"
+	ErrReadingFileCode              = "meshkit-11148"
+	ErrUnSupportedLayerTypeCode     = "meshkit-11149"
+	ErrGettingLayerCode             = "meshkit-11150"
+	ErrCompressingLayerCode         = "meshkit-11151"
+	ErrUnTaringLayerCode            = "meshkit-11152"
+	ErrGettingImageCode             = "meshkit-11153"
+	ErrValidatingImageCode          = "meshkit-11154"
+	ErrConnectingToRegistryCode     = "meshkit-11155"
+	ErrFileNotFoundCode             = "meshkit-11156"
+	ErrAuthenticatingToRegistryCode = "meshkit-11157"
+	ErrWriteFileCode                = "meshkit-11158"
+	ErrAddLayerCode = "meshkit-11159"
+	ErrTaggingPackageCode = "meshkit-11160"
```

Comment:
Do not add error codes yourself.  https://docs.meshery.io/project/contributing/contributing-error

Reply:
I just followed the sequence.  The error code previously mentioned are not present in here. https://docs.meshery.io/reference/error-codes#meshkit-library

Reply:
Yes, that's good that you ensured error codes are not repeated. But if you let the error-utility do that for you, you will see that the file will also be formatted and it might also happen that next error code to be used is the just next sequence number in the current error.go file.